psalm-baseline.xml

```diff
@@ -73,4 +73,9 @@
       <code>\Automattic\WooCommerce\Utilities\OrderUtil</code>
     </UndefinedClass>
   </file>
+  <file src="includes/compat/blocks/class-blocks-data-extractor.php">
+    <UndefinedClass occurrences="1">
+      <code>CheckoutSchema</code>
+    </UndefinedClass>
+  </file>
```

Comment:
I didn't know this was possible. We can probably clean up the inline ignores we have in `get_available_blocks` function. But that's not an expectation of this PR.

Reply:
Ah, yes 😄 thought of including it here as it was a minor change, but it's probably better to have it separated just to keep PRs focused.